corehq/apps/geospatial/dispatchers.py

```diff
@@ -0,0 +1,17 @@
+from corehq import toggles
+from corehq.apps.reports.dispatcher import (
+    ReportDispatcher,
+    cls_to_view_login_and_domain,
+)
+
+
+class CaseManagementMapDispatcher(ReportDispatcher):
+    prefix = 'geospatial'
+    map_name = 'GEOSPATIAL_MAP'
+
+    @cls_to_view_login_and_domain
+    def dispatch(self, request, *args, **kwargs):
+        return super(CaseManagementMapDispatcher, self).dispatch(request, *args, **kwargs)
+
+    def permissions_check(self, report, request, domain=None, is_navigation_check=False):
+        return toggles.GEOSPATIAL.enabled(domain)
```

Comment:
This can be removed after https://github.com/dimagi/commcare-hq/pull/33020/files#r1212829624.

Reply:
Updated :+1: